src/Maestro/tests/Maestro.ScenarioTests/ObjectHelpers/SubscriptionBuilder.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Microsoft.DotNet.Darc;
+using Microsoft.DotNet.Maestro.Client.Models;
+using Newtonsoft.Json.Linq;
+
+namespace Maestro.ScenarioTests.ObjectHelpers
+{
+    public class SubscriptionBuilder
+    {
+        /// <summary>
+        /// Creates a subscription object based on a standard set of test inputs
+        /// </summary>
+        public static Subscription BuildSubscription(string repo1Uri, string repo2Uri, string targetBranch, string channelName, string subscriptionId,
+            UpdateFrequency updateFrequency, bool batchable, List<string> mergePolicyNames, List<string> ignoreChecks)
+        {
+            Subscription expectedSubscription = new Subscription(Guid.Parse(subscriptionId), true, repo1Uri, repo2Uri, targetBranch);
+            expectedSubscription.Channel = new Channel(42, channelName, "test");
+            expectedSubscription.Policy = new SubscriptionPolicy(batchable, updateFrequency);
+
+            List<MergePolicy> mergePolicies = new List<MergePolicy>();
+
+            if (mergePolicyNames != null)
```

Comment:
I'd invert this if.  Right now the else is so far away that it's hard to keep it all in my head to figure out what it goes with, and what the behavior means.  In general, every level of indentation is some piece of context a reader/dev has to keep in their head when looking at the code.  So "less indented" translates to "less cognitive load" someone has to have to understand the code.  Less load means less likely to miss something and introduce a bug.

Also... I don't see where the "else" path returns, so I'm a little confused how this compiles...